UPGRADE.md

```diff
@@ -7,6 +7,13 @@ not skip minor/major releases while upgrading (fix releases can be skipped).
 The format is inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.8.x to 3.0.x
+
+### Before switching
+
+- The deprecated route `/lti-video/` is removed. You must move all your existing
+  link using this route to the route `/lti/videos/{id}` where `id` is the video id.
+
```

Comment:
If you're going to tag a 3.0 release, keep in mind that there are more deprecations to remove before that.